etl/command.py

```diff
@@ -80,12 +87,17 @@ def main_cli(
 ) -> None:
     _update_open_file_limit()
 
+    if upsert and not grapher:
+        click.echo("ERROR: --upsert also requires --grapher")
+        sys.exit(1)
```

Comment:
What if instead you just say:

```python
# enable grapher when called with --upsert
grapher = grapher or upsert
```

Then the usages become shorter too, just `etl --upsert ...`